docs/start-here.md

```diff
@@ -249,7 +249,7 @@ These are the queries in the development set of the MS MARCO passage ranking tes
 The first field is a unique identifier for the query (called the `qid`) and the second column is the query itself.
 These queries are taken from Bing search logs, so they're "realistic" web queries in that they may be ambiguous, contain typos, etc.
 
-Okay, let's now cross reference the `qid` with the relvance judgments, i.e., the qrels file:
+Okay, let's now cross-reference the `qid` with the relevance judgments, i.e., the qrels file:
```

Comment:
good catch!

Reply:
Thanks, I reverted IDs to ids, apologize for the mistake.

Reply:
no worries! no way you would have known...